lms/static/js/student_account/views/AccessView.js

```diff
@@ -190,7 +191,7 @@ var edx = edx || {};
             this.element.scrollTop( $anchor );
 
             // Update url without reloading page
-            History.pushState( null, document.title, '/account/' + type + '/' );
+            History.pushState( null, document.title, '/account/' + type + queryParams );
```

Comment:
Er, nevermind, forgot that this is just changing the history.  It might still be worth preserving the trailing slash if it makes the analysis in GA easier.